backend/testbackend.c

```diff
@@ -622,7 +622,7 @@ sigterm_handler(int sig)		/* I - Signal */
  * 'usage()' - Show usage information.
  */
 
-static void
+static void _CUPS_NORETURN
```

Comment:
The annotation should go on the prototype at the top of the source file, here and everywhere else...

Reply:
They are...